src/screens/Signup/styles.module.scss

```diff
@@ -1,4 +1,4 @@
-@import "scss/variables/_colors";
+@import 'scss/variables/_colors';
```

Comment:
again: `@import 'variables/colors';` does not work?

Reply:
No :cry: 
![Screenshot from 2021-05-14 12-32-37](https://user-images.githubusercontent.com/11763032/118293877-84c9e600-b4b0-11eb-9359-0cfcd1969268.png)

Maybe the project is missing some config?